sonaranalyzer-dotnet/src/AssemblyInfo.Shared.cs

```diff
@@ -21,9 +21,9 @@
 using System.Reflection;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("6.5.0")]
-[assembly: AssemblyFileVersion("6.5.0.0")]
-[assembly: AssemblyInformationalVersion("Version:6.5.0.0 Branch:improve-build-script Sha1:f3bccbf5bee16073c7ba18bc8dee5d3358bd21d8")]
+[assembly: AssemblyVersion("6.6.0")]
+[assembly: AssemblyFileVersion("6.6.0.0")]
+[assembly: AssemblyInformationalVersion("Version:6.6.0.0 Branch:improve-build-script Sha1:f3bccbf5bee16073c7ba18bc8dee5d3358bd21d8")]

```

Comment:
Guys we need to make sure we don't commit change (except version) on this file. It should look like "Version:X.X.X.X Branch:not-set Sha1:not-set"